src/app/routes/getInitialData/frontpage.test.js

```diff
@@ -0,0 +1,100 @@
+import baseUrl from './utils/getBaseUrl';
+import onClient from '../../lib/utilities/onClient';
+import fetchData from './utils/fetchData';
+import filterUnknownCpsTypes from '../../lib/utilities/preprocessor/rules/cpstypes';
+import filterEmptyGroupItems from '../../lib/utilities/preprocessor/rules/filterEmptyGroupItems';
+import applySquashTopstories from '../../lib/utilities/preprocessor/rules/topstories';
+
+const preprocessorRules = [
+  filterUnknownCpsTypes,
+  filterEmptyGroupItems,
+  applySquashTopstories,
+];
+
+process.env.SIMORGH_BASE_URL = 'https://www.SIMORGH_BASE_URL.com';
```

Comment:
this value is https://www.bbc.com in live, so this'd be https://www.https://www.bbc.com.com